src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -21,22 +21,25 @@ public class CommandResult {
 
     private final boolean isView;
 
+    private final boolean showEvent;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean isView) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean isView, boolean showEvent) {
```

Comment:
This was not the best implementation for isView, however it does works so its my bad  but adding onto the command result constructor may be devastating for larger projects